docs/development/releasing.md

```diff
@@ -243,19 +243,20 @@ Removing old .npmrc (default)
 Activating .npmrc "electron"
 ```
 
-The Electron account's credentials are kept by GitHub.
-"Electron - NPM" for the URL "https://www.npmjs.com/login".
+The Electron account's credentials are kept by GitHub in  a password manager.
+You'll also need to have access to an authenticator 2FA app with the appropriate
+OTP generator code to log in.
 ```sh
 $ npm login
-Username: electron
-Password:
+Username: electron-nightly
+Password: <This can be found under NPM Electron Nightly on LastPass>
 Email: (this IS public) electron@github.com
 ```
 
```

Comment:
This should include a note about setting the environment variable `ELECTRON_NPM_OTP` before running `npm run publish-to-npm`